zcash_client_sqlite/src/wallet/common.rs

```diff
@@ -148,9 +148,7 @@ where
                WHERE stx.block IS NOT NULL -- the spending tx is mined
                OR stx.expiry_height IS NULL -- the spending tx will not expire
              )
-             GROUP BY rn.id, t.txid, rn.{output_index_col},
-                      rn.diversifier, rn.value, {note_reconstruction_cols}, rn.commitment_tree_position,
-                      accounts.ufvk, rn.recipient_key_scope, t.mined_height"
+             GROUP BY rn.id"
```

Comment:
I added checks in the test that would catch a bug here if one existed, but it didn't because we were already grouping by `rn.id` as it's a returned column.